src/data/help.csv

```diff
@@ -0,0 +1,83 @@
+"さらに、<color=#ff9900>2人以上</color>のチームメンバーとアオハル特訓を発生させると\n<color=#ff9900>育成ウマ娘のトレーニングの効果もアップ</color>！\nまた、特訓に参加した<color=#ff9900>チームメンバーの数が増えるほど</color>、\n<color=#ff9900>トレーニングで得られる基礎能力やスキルPtが大きくなります</color>。\nただし、その分<color=#ff9900>消費する体力も大きくなる</color>ため注意しましょう。","In addition, a special Aoharu training session <color=#ff9900>with 2 or more</color> team members, will <color=#ff9900>boost your Uma Musume's training efficiency as well\nAlso, <color=#ff9900>the more team members</color> participate in the training, <color=#ff9900>the more basic abilities and skill points you will gain</color> from the training\nHowever, be aware that <color=#ff9900>the amount of energy you consume will also increase</color>."
+"アオハル特訓を何回も行ったチームメンバーは、\n一度だけ<color=#ff9900>アオハル魂爆発</color>が発生することがあります！\n発生すると、そのチームメンバーの<color=#ff9900>基礎能力が大きく上がり</color>、\n<color=#ff9900>育成ウマ娘も成長しスキルのヒントを取得</color>します！\n<color=#ff9900>アオハル魂爆発を多く発生させる</color>ことが、攻略のポイントです！","Team members who have gone through special Aoharu Training many times can have a single <color=#ff9900>Aoharu Soul Explosion</color>\nWhen this happens, the team member's <color=#ff9900>basic abilities will increase greatly, and the trained Uma-Musume will also grow and get skill hints</color>\nThe key to success is to <color=#ff9900>generate as many Aoharu Soul Explosions as possible</color>!"
+"現在持っているアイテムを確認することができます。\nまた、消費アイテムを使用することもできます。\n一部のアイテムはマニーに変換することができます。\n<color=#ff9900>変換したアイテムは元に戻すことができません</color>ので、変換の際には注意しましょう。","You can check the items you currently have.\nYou can also use consumable items.\nSome items can be converted to Manny.\n<color=#ff9900>Converted items cannot be undone</color>, so be careful when converting."
+"チームメンバーが成長することで、<color=#ff9900>チーム総合力</color>が上がり、\nたづなさんから差し入れが貰えます。また、チームの各基礎能力が\n上がることで、<color=#ff9900>トレーニングLvも上昇</color>します。\n<color=#ff9900>チームの強さがトレーニングでの育成ウマ娘の成長に繋がる</color>ので、\n積極的にチームを鍛えていきましょう！","As your team members grow, your <color=#ff9900>overall team strength</color> will increase and you will receive gifts from Tazuna. Also, as each team member's basic ability increases, their <color=#ff9900>Training Lv will increase</color>\n<color=#ff9900>The strength of your team will lead to the growth of the Uma-Musume you train</color>, so be proactive and train your team!"
+"<color=#ff9900>チーム称号</color>とは、チーム総合力とチームランキングを\n<color=#ff9900>総合した評価</color>です。\n持っている<color=#ff9900>チーム称号が優れているほど</color>、\n<color=#ff9900>エンディングでウマ娘が大きく成長</color>します！\nアオハル杯を勝ち抜き、最強のチーム称号を手に入れましょう！","The <color=#ff9900>team title</color> is an <color=#ff9900>evaluation of your overall</color> team strength and team ranking\n<color=#ff9900>The better the team title you have, the more your horse girl will grow in the ending</color>\nLet's win Aoharu Hai and get the strongest team title!"
```

Comment:
Holy! Have you checked how these looked like ingame? Unfortunately with the current text replacement method longer text will get cut off or look very weird if it's too long.

Reply:
I'm not sure how to consistently check these.
They are long, but there is quite a bit of space and there seem to be no issues with the text rendering that I could see.
Some do run off the page but removing newlines would probably help that.
Besides that, I'm working off the idea that having the text in English takes precedence over neat display, as long as it's not a complete mess.
Unless someone can/wants to check and edit the lengths on these I will remove the newlines in a separate commit for now.

Reply:
![image](https://user-images.githubusercontent.com/7202881/131828728-9eed85c0-5b34-4348-91ff-2a9604e49560.png)

Don't think that these should be much of a problem, this is the worst one I was able to find (wasn't looking that much tho). At least when it comes to space constraints, haven't actually checked the translations.

Reply:
Just checked it, it looks [amazing](https://user-images.githubusercontent.com/25855364/131909048-d70bbb02-d37b-4000-8b17-c8c2fa6ded40.png)!  
I think it's fine for now; we can refine those that are too long in the future.

Reply:
Oh yea that is absolutely splendid.